ethereum-types/src/hash.rs

```diff
@@ -136,20 +153,28 @@ mod tests {
 
 	#[test]
 	fn test_parse_0x() {
-		assert!("0x0000000000000000000000000000000000000000000000000000000000000000".parse::<H256>().is_ok())
+		assert!("0x0000000000000000000000000000000000000000000000000000000000000000"
+			.parse::<H256>()
+			.is_ok())
 	}
 
 	#[test]
 	fn test_serialize_invalid() {
-		assert!(ser::from_str::<H256>("\"0x000000000000000000000000000000000000000000000000000000000000000\"")
-			.unwrap_err()
-			.is_data());
-		assert!(ser::from_str::<H256>("\"0x000000000000000000000000000000000000000000000000000000000000000g\"")
-			.unwrap_err()
-			.is_data());
-		assert!(ser::from_str::<H256>("\"0x00000000000000000000000000000000000000000000000000000000000000000\"")
-			.unwrap_err()
-			.is_data());
+		assert!(ser::from_str::<H256>(
+			"\"0x000000000000000000000000000000000000000000000000000000000000000\""
+		)
+		.unwrap_err()
+		.is_data());
```

Comment:
doesn't seem like an improvement to me TBH

Reply:
If the string is a few chars shorter it does a great job. I've queried this behaviour: https://github.com/rust-lang/rustfmt/issues/4912